### DISCUSSION TITLE
Cron job is not showing up in preview, it is only works in production?

### DISCUSSION THREAD

**Discussion by avarajar** [vercel/community#1928] [Help] [answered]
### Summary

I deployed my Vercel project in a preview/development mode, but I can't see the cron job in the dashboard.. It is only works in production?

**Comment by leerob** [comment, marked as answer]
This is intended, Vercel Cron only runs on production environments.

https://vercel.com/docs/cron-jobs#:~:text=Preview%20deployments%20are%20ignored.
